### ISSUE TITLE
if a column name in the header row contains a period, the values in the column will be null

### ISSUE THREAD

**Issue by judell** [turbot/steampipe-plugin-csv#24]
Given this in `seitz.csv`

```
"subscriptions.id","subscriptions_plan_id","subscriptions_plan_quantity"
a,b,1
d,e,2
```

The result for `select * from csv.seitz`:

```
+------------------+-----------------------+-----------------------------+
| subscriptions.id | subscriptions_plan_id | subscriptions_plan_quantity |
+------------------+-----------------------+-----------------------------+
| <null>           | b                     | 1                           |
| <null>           | e                     | 2                           |
+------------------+-----------------------+-----------------------------+
```

**Comment by Subhajit97** [comment]
hi @judell, we have looked into it and seems like it needs an update in `turbot/go-kit/helpers`.
https://github.com/turbot/go-kit/issues/21
We will provide an update here, once the changes are available.

**Comment by Subhajit97** [comment]
@judell We have merged the required changes, and the fix will be available in the next plugin release. We will provide an update when it is available.

**Comment by Subhajit97** [comment]
Hi @judell, this bug should now be fixed in CSV plugin  `v0.3.2`. If you still see the same behaviour after updating, please comment and re-open this issue; thanks!
